services/web/server/tests/unit/isolated/test_garbage_collector_core.py

```diff
@@ -80,3 +84,56 @@ async def test_regression_project_id_recovered_from_the_wrong_data_structure(
         },
         currently_opened_projects_node_ids={},
     )
+
+
+async def test_remove_single_service_if_orphan_service_is_waiting_manual_intervention(
+    faker: Faker,
+    mocker: MockerFixture,
+    aioresponses_mocker: AioResponsesMock,
+):
+
+    base_module = "simcore_service_webserver.garbage_collector_core"
```

Comment:
Perhaps a bit of refactoring here (e.g. common fixtures) might help for the next time we add tests on the GC. :-)